Code/Tools/ProjectManager/Source/TemplateButtonWidget.cpp

```diff
@@ -6,16 +6,44 @@
  */
 
 #include <TemplateButtonWidget.h>
+#include <ProjectManagerDefs.h>
 
 #include <QVBoxLayout>
 #include <QLabel>
 #include <QPixmap>
+#include <QIcon>
 #include <QAbstractButton>
 #include <QStyle>
 #include <QVariant>
 
 namespace O3DE::ProjectManager
 {
+    TemplateImageOverlay::TemplateImageOverlay(const QString& imagePath, QWidget* parent)
+        : QLabel(parent)
+    {
+        QVBoxLayout* vLayout = new QVBoxLayout();
+        vLayout->setSpacing(0);
+        vLayout->setContentsMargins(0, 0, 0, 0);
+        setLayout(vLayout);
+
+        setObjectName("templateImage");
+        setPixmap(QPixmap(imagePath).scaled(
```

Comment:
Any way we can use a QIcon here to keep the vector graphics and don't convert to a pixel-based image and resize it?

Reply:
This image is an png, the below overlay label below is the svg.